lib/dradis/plugins/settings.rb

```diff
@@ -44,6 +44,15 @@ def is_default?(key, value)
       value.to_s == @default_options[key.to_sym].to_s
     end
 
+    def engine_enable
```

Comment:
https://github.com/rubocop/ruby-style-guide#bool-methods-qmark
```suggestion
    def enabled?
```